src/pages/ScholarshipForm.jsx

```diff
@@ -38,9 +63,18 @@ function ScholarshipForm() {
         {requiredTextField('name', 'Scholarship Name')}
         {requiredTextField('description', 'Description')}
         {requiredTextField('website', 'Website')}
-        <Button variant="contained" color="primary">
-          Submit
-        </Button>
+        {scholarshipTypeField('scholarshipType', 'Scholarship Type', [
+          'unknown',
+          'academic',
+          'sports',
+          'community',
+          'organization',
+        ])}
```

Comment:
Two things:
1. These values should come from an enum type. 
2. I'm not sure if these types make the most sense.

Reply:
I'm going to make a PR adding the enums to a file so that I can use them here.
What should the types be? There are the types from the model.

Reply:
The enum change doesn't necessarily need to be a separate PR if the changes are small enough and no other changes depend on it